docs/UserGuide.adoc

```diff
@@ -172,35 +172,83 @@ Lists all the commands that you have entered in reverse chronological order. +
 Format: `history` +
 Alias: `hist`
 
-=== Creating a new journal entry : `new`
+[NOTE]
+====
+Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
+====
+
+=== Viewing a timetable : `tview`
+
+Shows the current saved timetable. +
+Format: `tview` +
+Alias: `tv`
+
+=== Adding a timetable : `tadd`
+
+Imports a timetable from NUSMODS website. +
+Format: `tadd` +
+Alias: `ta`
+
+=== Replacing a timetable : `treplace`
+
+Replaces the current timetable with an updated version. +
+Format: `treplace` +
+Alias: `tr`
+
+=== Viewing all journal entries : `jview`
+
+View all journal entries. +
+Format: `jview` +
+Alias: `jv`
+
+=== Creating a new journal entry : `jnew`
+
+Creates a new window that allows the user to input and save journal entries. +
+Format: `jnew` +
+Alias: `jn`
+
+=== Deleting a journal entry : `jdelete`
+
+Deletes a journal entry on the specified date. +
+Format: `jdelete` +
+Alias: `jd`
+
+=== Filtering journal entries by date : `jfilter`
+
+Filters journal entries by the specified internal. +
+Format: `jfilter` +
+Alias: `jf`
 
```

Comment:
not really impt imo this jfilter

what does specified internal mean?

Reply:
i think he meant specific?

Reply:
specified interval* haha sorry had typo